src/main/webapp/quiz.html

```diff
@@ -62,6 +62,10 @@ <h2 id="quiz-title">Style Quiz</h2>
             <button id="next">Next</button>
 
             <!-- Submit Quiz Button -->
+            <!-- <form id="answers">
```

Comment:
This seems unrelated to your PR?

Reply:
Oh I think I committed the on accident. It's no different than what's already in the repo.